changelog/fragments/ansible-limits.yaml

```diff
@@ -0,0 +1,50 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible-based operators, add default resource limits for the manager.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Header text for the migration section
+      body: >
+        Add into the config/manager/manager.yaml` the following limits scaffold by default:
+
+        ```yaml
+        ...
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 250m
```

Comment:
This is really a lot for a controller and would be an issue for deploying such operators on small clusters that are used for instance on developer machines, retail shops, factory floors, etc
I would propose 25m if ansible operators are using a lot of CPU. Typically well behaving go controllers use ~10m in average on mid-size clusters